modules/microshift-firewall-verify-settings.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-firewall-verifying-settings_{context}"]
+== Verifying firewall settings
```

Comment:
I think that the first header is supposed to be H1, so "=" instead of "==". You may see some asciibinder errors if you don't have "=" at the beginning of the module. If you need this module nested you can leveloffset+2 in the assembly where it's included. I've noticed this in a few modules for this PR so this is more of a global comment to check that the first header in your modules is a level 1 "=".